### ISSUE TITLE
The future of this project

### ISSUE THREAD

**Issue by claudiosegala** [claudiosegala/regalia#1]
Will we continue this? What are the plans? What do we want to improve? Will we try to put on steam?

**Comment by ramkury** [comment]
From my end, I would mostly like to improve how the game interacts with the gamepads:
* Allowing for hot swapping controllers (that is, the game should be able to handle controllers connecting or disconnecting while it's running)
* Adding support for haptic feedback

If our artists are willing to keep working on this project, we could add new backgrounds, tile sets, tile maps and even the remaining playable characters (Alquimista and Sebastião).
Also, maybe we should work on letting the players choose a stage for the battle (instead of always choosing at random) and assigning specific backgrounds to each tile map.

We are very, very far away from a stage where we should even consider publishing the game on any platform such as Steam. That should not be a concern right now.

**Comment by claudiosegala** [comment]
Sounds fair. From my end, I would like to implement new features like:
+ New dynamics (ex: Teleportation, New Powers)
+ Add a way to rewind the game for when the last player dies (maybe)

We have to check if Dedé and Nathan want to continue. I think that Dedé would like to finish the book at least and Nathan maybe would like to help. But we gotta check with them.

**Comment by ramkury** [comment]
Do you mean rewind, like in Forza, or simply show a replay of the last few seconds, like in Super Meat Boy and Towerfall?

**Comment by claudiosegala** [comment]
Yeah, like that or something that make the game a little more interesting.

**Comment by claudiosegala** [comment]
@ramkury @gustavohfc @hectorid, I just discovered that I won't be able to work on this project until I finish my TCC. As soon as I finish it, I want to go back improving this project.
